dbt_subprojects/solana/models/_sector/dex/pancakeswap/schema.yml

```diff
@@ -0,0 +1,120 @@
+  - name: pancakeswap_v3_base_trades
+    meta:
+      blockchain: solana
+      contributors: [chef_butterbeer]
+    config:
+      tags: ["solana", "dex"]
+    description: >
+      all raw pancakeswap_v3 CLAMM dex trades on Solana
+    data_tests:
+      - check_columns_solana_dex_trades
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            [
+              "tx_id",
+              "outer_instruction_index",
+              "inner_instruction_index",
+              "tx_index",
+              "block_month",
+            ]
+
+  - name: pancakeswap_v3_trades
+    meta:
+      blockchain: solana
+      contributors: [chef_butterbeer]
+    config:
+      tags: ["solana", "dex"]
+    description: >
+      all pancakeswap_v3 CLAMM dex trades on Solana
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "Blockchain which the DEX is deployed"
+      - &project
+        name: project
+        description: "Project name of the DEX"
+      - &version
+        name: version
+        description: "Version of the contract built and deployed by the DEX project"
+      - &block_month
+        name: block_month
+        description: "UTC event block month of each DEX trade"
+      - &block_time
+        name: block_time
+        description: "UTC event block time of each DEX trade"
+      - &block_slot
+        name: block_slot
+        description: "block slot of each DEX trade"
+      - &trade_source
+        name: trade_source
+        description: "Was the trade a direct call to the dexor did it go through another program like Jupiter (Dex Aggregator)"
+      - &token_bought_symbol
+        name: token_bought_symbol
+        description: "Token symbol for token bought in the trade"
+      - &token_sold_symbol
+        name: token_sold_symbol
+        description: "Token symbol for token sold in the trade"
+      - &token_pair
+        name: token_pair
+        description: "Token symbol pair for each token involved in the trade"
+      - &token_bought_amount
+        name: token_bought_amount
+        description: "Value of the token bought at time of execution in the original currency"
+      - &token_sold_amount
+        name: token_sold_amount
+        description: "Value of the token sold at time of execution in the original currency"
+      - &token_bought_amount_raw
+        name: token_bought_amount_raw
+        description: "Raw value of the token bought at time of execution in the original currency"
+      - &token_sold_amount_raw
+        name: token_sold_amount_raw
+        description: "Raw value of the token sold at time of execution in the original currency"
+      - &amount_usd
+        name: amount_usd
+        description: "USD value of the trade at time of execution"
+        data_tests:
+          - dbt_utils.accepted_range:
+              max_value: 1000000000 # $1b is an arbitrary number, intended to flag outlier amounts early
+              severity: warn
```

Comment:
```suggestion
```

Reply:
should only need this test in upstream `dex_solana_trades` lineage, having here is duplicating for no reason